sklearn/utils/estimator_checks.py

```diff
@@ -272,6 +272,8 @@ def _yield_all_checks(name, estimator):
     yield check_dict_unchanged
     yield check_dont_overwrite_parameters
     yield check_fit_idempotent
+    if not tags["no_validation"]:
```

Comment:
does it make sense to make sure the `n_features_in_` is `None` or doesn't exist
if the `no_validation` tag is set?

Reply:
I would say no, since you could have a meta estimator that delegates the validation to its estimators, and still have the attribute?

Reply:
Ok.